### PR TITLE
Add `policy` and `dynamic-config` sub-commands to `aperturectl apply`

### PR DESCRIPTION
**Pull request by hasit**
##### Checklist

- [x] Tested in playground or other setup
- [ ] Screenshot (Grafana) from playground added to PR for 15+ minute run
- [x] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

- - -
This change is [Reviewable](https://reviewable.io/reviews/fluxninja/aperture/1309)

### CODE REVIEW

**Comment by hasit** [comment]
[Screencast from 02-06-2023 12:14:28 PM.webm](https://user-images.githubusercontent.com/1553055/217075923-73a26e81-5b97-4169-8cb6-79afbd859999.webm)

**Comment by hasit** [comment]
demo

File: cmd/aperturectl/cmd/blueprints/root.go [hunk 1]
```diff
@@ -90,7 +96,7 @@ Use this command to pull, list, remove and generate Aperture Policy resources us
 		lock = flock.New(filepath.Join(blueprintsDir, lockFilename))
 
 		// pull the latest blueprints based on skipPull and whether cmd is remove
-		if !skipPull && cmd.Use != "remove" {
+		if !skipPull && cmd.Use != "remove" && cmd.Use != "list" {
```
Comment (harjotgill) [inline cmd/aperturectl/cmd/blueprints/root.go:99]: On list we have to pull.

**Review by harjotgill — APPROVED** [review]

File: cmd/aperturectl/cmd/blueprints/root.go (hunk 1 above)
Comment (hasit) [reply cmd/aperturectl/cmd/blueprints/root.go]: [f201d1b](https://github.com/fluxninja/aperture/pull/1309/commits/f201d1bf88eb6c33d2940fd86578acac508ce995) addresses this

**Pull request merged**